### ISSUE TITLE
Package removed from CRAN

### ISSUE THREAD

**Issue by vonGameTheory** [dalpozz/unbalanced#6]
This package was removed from CRAN a few weeks ago in case you are not aware.  (Reason was undeliverable email)

**Comment by dalpozz** [comment]
yes, I'm aware, I've been trying to resubmit a new package but I'm not successful yet and don't have much time to look into it

**Comment by ghost** [comment]
Is there an ETA? I heard this package had a great "Racing" algorithm. Thanks!
